lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -47,6 +49,7 @@ def tick_update_position(tick, tickxs, tickys, labelpos):
 
     tick.tick1On, tick.tick2On = True, False
     tick.tick1line.set_linestyle('-')
+    tick.tick1line.set_linewidth(rcParams['grid.linewidth'])
```

Comment:
Historically, we have been treating the z-axis like an x-axis in terms of
style settings. So, I would do it as searching for its axis parameter, and
fail back to the x-axis parameter.

On Wed, Apr 13, 2016 at 10:03 AM, Michael Droettboom <
notifications@github.com> wrote:

> In lib/mpl_toolkits/mplot3d/axis3d.py
> <https://github.com/matplotlib/matplotlib/pull/6292#discussion_r59553056>:
>
> > @@ -47,6 +49,7 @@ def tick_update_position(tick, tickxs, tickys, labelpos):
> >
> >      tick.tick1On, tick.tick2On = True, False
> >      tick.tick1line.set_linestyle('-')
> > +    tick.tick1line.set_linewidth(rcParams['grid.linewidth'])
>
> You are right -- xtick.major.width would be better. But what about ztick
> -- we don't have rcParams for that. Should we make them?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/6292/files/6cd53dd001d4952fe398698d3aa043f8ae8b9a18#r59553056>
>